archiver/archiver.py

```diff
@@ -178,18 +178,25 @@ def from_uuid(cls, ingest_api, biomaterial_uuid):
 
 
 class IngestAccession:
```

Comment:
could be renamed IngestAccessioner class, which is the component who knows where in the metadata an accession should be put.